src/index.html

```diff
@@ -7,6 +7,330 @@
   <title>基本コーディング</title>
 </head>
 <body>
+  <header class="l-header">
+    <nav class="l-header__nav">
+      <div class="l-header__pc-items">
+        <ul class="l-header__items">
+          <li class="l-header__item"><a href="">ギズジョブ医療事務とは</a></li>
+          <li class="l-header__item"><a class="is-search" href="">こだわり検索</a></li>
+          <li class="l-header__item"><a class="is-book" href="">ギズジョブ医療事務コラム</a></li>
+        </ul>
+        <ul class="l-header__buttons">
+          <li class="l-header__button is-heart"><a href="">気になるリスト</a></li>
+          <li class="l-header__button is-clock"><a href="">閲覧履歴</a></li>
+          <li class="l-header__button is-camera"><a href="">検索履歴</a></li>
+        </ul>
+      </div>
+      <div class="l-header__sp-items">
+        <button class="l-header__hamburger">
+          <span></span>
+          <span></span>
+          <span></span>
+        </button>
+      </div>
+    </nav>
+  </header>
 
+  <main class="main">
+    <div class="keyvisual">
+      <img src="./img/picture/グループ 23.png" class="nurse-photo">
```

Comment:
[must]

`npm run dev` をしてブラウザを確認したところ以下のようなエラーが表示されていました。

![スクリーンショット 2024-03-05 17 19 50](https://github.com/gizumo-education/front-coding/assets/48352529/926bdf36-832a-45c6-bdb9-64d386dde447)

```suggestion
      <img src="/img/picture/グループ 23.png" class="nurse-photo">
```
 
このエラー1つ1つは上記の `./img ...` としているところを `/img ...` に修正すれば解消されるかと思います。

なので、他の画像パスを使用している箇所に関しても修正してもらってエラーが消えた段階で再度レビュー依頼をお願いします。
必ず PR を出す前にほんの少しでも修正したら画面上問題がないか確認してからレビュー依頼をするようにしましょう